.github/workflows/ci.yml

```diff
@@ -57,7 +57,64 @@ jobs:
         id: test-action
         uses: ./
         with:
-          pkl-version: 0.25.1
+          pkl-version: 0.26.0
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Confirm download
+        run: pkl --version
+
+  test-action-macos-amd64:
+    name: GitHub Actions Test MacOS amd64
+    runs-on: macos-latest
```

Comment:
`macos-latest` is equivalent to `macos-14`, probably want to check `macos-13`

https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-private-repositories

Reply:
This catch! Thanks for this. I updated the branch. Should work now ✅